rasa/core/test.py

```diff
@@ -667,6 +638,16 @@ def _collect_action_executed_predictions(
                     "training stories and retrain."
                 )
             raise WrongPredictionException(error_msg)
+    elif previously_predicted_action_unlikely_intent:
```

Comment:
is this possible given that we already have `predicted_action_unlikely_intent and predicted_action != expected_action` in the `or` of the associated `if`?

Reply:
It is possible. `prev_action_unlikely_intent` is different from `predicted_action_unlikely_intent and predicted_action != expected_action` (see comment above)